docs/AboutUs.md

```diff
@@ -46,8 +46,8 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[github](https://github.com/stein414)]
 [[portfolio](team/ian.md)]
 
-* Role: TBD
-* Responsibilities: TBD
+* Role: Code Quality, VS Code Expert
+* Responsibilities: Issues
```

Comment:
Code Quality sounds more like a responsibility. You could put "Developer, VS Code Expert" for your role.